--- libassuan-0.6.2/src/libassuan.m4.orig	2003-04-29 12:49:57.000000000 +0200
+++ libassuan-0.6.2/src/libassuan.m4	2003-12-18 20:34:23.344048576 +0100
@@ -14,7 +14,7 @@
 dnl                   [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND ]]])
 dnl Test for libassuan and define LIBASSUAN_CFLAGS and LIBASSUAN_LIBS
 dnl
-AC_DEFUN(AM_PATH_LIBASSUAN,
+AC_DEFUN([AM_PATH_LIBASSUAN],
 [ AC_ARG_WITH(libassuan-prefix,
             AC_HELP_STRING([--with-libassuan-prefix=PFX],
                            [prefix where LIBASSUAN is installed (optional)]),
