--- libassuan-3.0.0/configure.ac.orig	2024-06-23 20:03:45.691847685 +0200
+++ libassuan-3.0.0/configure.ac	2024-06-23 20:05:14.248034601 +0200
@@ -245,11 +245,6 @@ have_dosish_system=no
 have_w32_system=no
 have_w64_system=no
 case "${host}" in
-    *-linux*)
-        if test "$GCC" = yes; then
-          CFLAGS="$CFLAGS -fPIC -DPIC"
-        fi
-        ;;
     x86_64-*mingw32*)
         have_dosish_system=yes
         have_w32_system=yes
