--- libassuan-2.0.1/configure.ac.orig	2010-08-09 12:29:56.000000000 +0200
+++ libassuan-2.0.1/configure.ac	2010-08-09 20:10:45.442823295 +0200
@@ -63,7 +63,10 @@
 AB_INIT
 
 AC_GNU_SOURCE
-
+AH_TEMPLATE([_ALL_SOURCE], [All extensions])
+AH_TEMPLATE([_GNU_SOURCE], [GNU extensions])
+AH_TEMPLATE([_POSIX_PTHREAD_SEMANTICS], [POSIX pthread semantics])
+AH_TEMPLATE([_TANDEM_SOURCE], [Tandem extensions])
 
 AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
@@ -154,11 +157,6 @@
 have_w32_system=no
 have_w32ce_system=no
 case "${host}" in
-    *-linux*)
-        if test "$GCC" = yes; then
-          CFLAGS="$CFLAGS -fPIC -DPIC"
-        fi
-        ;;
     *-mingw32ce*)
         have_dosish_system=yes
         have_w32_system=yes
