--- libassuan-0.6.1/configure.ac.orig	2003-11-17 18:55:56.000000000 +0100
+++ libassuan-0.6.1/configure.ac	2003-11-17 19:07:33.848461632 +0100
@@ -90,7 +90,7 @@
 AC_CHECK_FUNCS(funopen)
 if test $ac_cv_func_funopen != yes; then
   # No funopen but we can implement that in terms of fopencookie.
-  AC_CHECK_FUNCS(fopencookie, AC_LIBOBJ(funopen), AC_MSG_ERROR([[
+  AC_CHECK_FUNCS(fopencookie, [AC_LIBOBJ(funopen)], AC_MSG_ERROR([[
 No implementation of fopencookie or funopen available.
 ]]))
 fi
