--- libassuan-2.0.2/configure.ac.orig	2011-06-16 13:47:56.000000000 +0200
+++ libassuan-2.0.2/configure.ac	2011-06-18 08:36:25.247982816 +0200
@@ -172,11 +175,6 @@
 have_w64_system=no
 have_w32ce_system=no
 case "${host}" in
-    *-linux*)
-        if test "$GCC" = yes; then
-          CFLAGS="$CFLAGS -fPIC -DPIC"
-        fi
-        ;;
     x86_64-*mingw32*)
         have_dosish_system=yes
         have_w32_system=yes
